Library/src/main/java/com/shopify/testify/internal/processor/BitmapExtentions.kt

```diff
@@ -13,7 +15,17 @@ fun ParallelPixelProcessor.TransformResult.createBitmap(): Bitmap {
     )
 }
 
-var numberOfCores = Runtime.getRuntime().availableProcessors()
+private val numberOfAvailableCores = Runtime.getRuntime().availableProcessors()
+
+@VisibleForTesting
+var maxNumberOfChunkThreads = numberOfAvailableCores
```

Comment:
The default is to create a coroutine threadpool matching the number of processor cores on your device. But, for testing, I wanted to be able to override this.